checkstyle-config.xml

```diff
@@ -48,6 +48,7 @@
         <module name="FinalClass"/>
         <module name="HiddenField">
             <property name="ignoreConstructorParameter" value="true"/>
+            <property name="ignoreSetter" value="true"/>
```

Comment:
Any idea why this is not enough to prevent the following errors?

```
[ERROR] src/test/java/nl/jqno/equalsverifier/integration/extended_contract/MapEntrySubclassTest.java:[59,29] (coding) HiddenField: 'value' hides a field.
[ERROR] src/test/java/nl/jqno/equalsverifier/integration/extended_contract/MapEntrySubclassTest.java:[99,29] (coding) HiddenField: 'value' hides a field.
```

Reply:
I honestly don't know... maybe you can just add `// CHECKSTYLE OFF: HiddenField` to the file in question though?

Reply:
Yep, that did the trick. Thanks!